code/datums/observation/observation.dm

```diff
@@ -62,76 +62,241 @@
 /decl/observ
 	var/name = "Unnamed Event" // The name of this event, used mainly for debug/VV purposes. The list of event managers can be reached through the "Debug Controller" verb, selecting the "Observation" entry.
 	var/expected_type = /datum // The expected event source for this event. register() will CRASH() if it receives an unexpected type.
-	var/list/event_sources     // Associate list of event sources, each with their own associate list. This associate list contains an instance/proc pair to call when the event is raised.
+	var/list/event_sources     // Associative list of event sources, each with their own associative list. This associative list contains an instance/proc pair to call when the event is raised.
 	var/list/global_listeners  // Associative list of instances that listen to all events of this type (as opposed to events belonging to a specific source) and the proc to call.
 
 /decl/observ/New()
 	all_observable_events.events += src
-	event_sources = list()
-	..()
+	return ..()
```

Comment:
Well, that's news to me.  I will update the PR in the morning.
On Feb 9, 2016 10:38 PM, "PsiOmegaDelta" <notifications@github.com> wrote:

> In code/datums/observation/observation.dm
> <https://github.com/PsiOmegaDelta/Baystation12/pull/37#discussion_r52416884>
> :
>
> >  	var/list/global_listeners  // Associative list of instances that listen to all events of this type (as opposed to events belonging to a specific source) and the proc to call.
> >
> >  /decl/observ/New()
> >  	all_observable_events.events += src
> > -	event_sources = list()
> > -	..()
> > +	return ..()
>
> According to ref New() should never return.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PsiOmegaDelta/Baystation12/pull/37/files#r52416884>.
>